public/_headers

```diff
@@ -14,6 +14,8 @@
 
 /headerMenu.json
     Access-Control-Allow-Origin: *
+    Access-Control-Allow-Headers: content-type
```

Comment:
Do we actually need these? Like, does it not work if these headers are not present?